hub/apps/how-tos/uno-multiplatform.md

```diff
@@ -0,0 +1,119 @@
+
+    :::image type="content" source="images/hello-world/manage-extensions.png" alt-text="Visual Studio Menu bar item that reads manage extensions":::
+
+2. In the Extension Manager expand the **Online** node and search for `Uno`, install the `Uno Platform` extension or download it from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=unoplatform.uno-platform-addin-2022), then restart Visual Studio.
+
+    :::image type="content" source="images/hello-world/uno-extensions.PNG" alt-text="Visual Studio Menu bar item that reads manage extensions":::
+
+## Create an application
+
+Now that we are ready to create a multi-platform application, the approach we'll take is to create a new Uno Platform application, and then copy the existing WinUI 3 project items into it. This will allow you to reuse the existing codebase, and then you can add the necessary changes over time to make new functionality work on other platforms. This approach is also useful if you have an existing application that you want to port to other platforms.
+
+Soon enough, you will be able to reap the benefits of this approach, as you can target more platforms with a familiar XAML flavor and the codebase you already have.
+
+To create an Uno Platform app:
+
+1. Create a new C# solution using the **Uno Platform App** template, from Visual Studio's **Start Page**
+2. Choose a base template to take your hello world application multi-platform
+
+    :::image type="content" source="images/hello-world/vsix-new-project-options.png" alt-text="Uno solution template for project startup type":::
+
+3. You can optionally choose to customize your app based on the sections on the left side:
+    - **Framework** allows to choose which `TargetFramework` your app will use. [.NET 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) is a commonly appropriate choice.
+    - **Platforms** provides a list of platforms your application will support. You can choose to support all platforms, or a subset of them
+    - **Presentation** gives a choice about using MVVM (e.g. [MVVM Toolkit](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/mvvm/)) or Uno Platform's MVUX and Feeds
+    - **Projects** gives the ability to add a Server project for APIs and hosting for the WebAssembly project
+    - **Testing** provides Unit Testing and [UI Testing projects](https://github.com/unoplatform/Uno.UITest)
+    - **Features** provides support for WebAssembly PWA and optional VS Code support files
+    - **Extensions** allows to choose for additional [Uno.Extensions](https://github.com/unoplatform/uno.extensions) to kickstart your app faster
+    - **Application** sets the App ID for relevant platforms, used when publishing on various app stores.
+    - **Theme** gives the ability to change between Fluent and Material themes
+
+4. Click the create button
```

Comment:
Capital C in Create

Reply:
If you're referencing the name of UI elements, I recommend making them **bold** as well.